CHANGELOG.md

```diff
@@ -0,0 +1,79 @@
+v 0.3 API changes:
+
+C library:
+
+Changes in ccl_core.c:
+
+Old call signature:
+ccl_parameters ccl_parameters_create(double Omega_c, double Omega_b, double Omega_k,
+				     double N_nu_rel, double N_nu_mass, double mnu,
+				     double w0, double wa, double h, double norm_pk,
+				     double n_s, double bcm_log10Mc, double bcm_etab, double bcm_ks,
+				     int nz_mgrowth,double *zarr_mgrowth,
+					 double *dfarr_mgrowth, int *status)
+					 
+New call signature:
+ccl_parameters_create(double Omega_c, double Omega_b, double Omega_k,
```

Comment:
Sounds good, I wasn't sure what to do since this is the first time we've changed the API since the new API-change-documentation guidance so I was just winging it. I've made it shorter.